.travis.yml

```diff
@@ -1,3 +1,4 @@
+# Travis CI configuration settings
 language: ruby
 sudo: false
```

Comment:
<a href="https://www.youtube.com/watch?v=fq3abPnEEGE"><img src="http://i1.kym-cdn.com/photos/images/newsfeed/001/029/129/650.gif" width="256" alt="It is a mystery" /></a>